annaeliana7/Mountain-Formation
# Final Touches

## What
This pull request contains the following changes:

- Refactored carousel images and added 7 additional images
- Updated README with necessary information

## Why
The changes need to be approved through a pull request before they can be merged.

## How
How can it be tested?

- Verify that the carousel images have been refactored and that 7 additional images have been added.
- Check the README file for the updated information.

## Project Demo
![project demo-2](https://user-images.githubusercontent.com/112557951/215263784-d1265c61-8966-45da-a601-49e01d9be0ed.jpg)


### Changes details
- Detail one: Refactor carousel images and add 7 additional image
- Detail two: Update README with necessary information

## Missed anything?
- [x] Explained the purpose of this PR.
- [x] Self reviewed the PR.
- [ ] Added or updated test cases.

## Closing note
This PR has been tested, reviewed and ready for final review.
